OpenSim/Simulation/Model/Frame.h

```diff
@@ -193,6 +206,41 @@ OpenSim_DECLARE_ABSTRACT_OBJECT(Frame, ModelComponent);
     */
     SimTK::Vec3 findLocationInAnotherFrame(const SimTK::State& state, 
                     const SimTK::Vec3& point_F, const Frame& otherFrame) const;
+
+    /**
+    Take a point located and expressed in this frame (F) and determine
+    its location expressed in Ground (G). This method is equivalent to
+    `findLocationInAnotherFrame()` where `otherFrame` is always Ground.
+
+    @param state       The state of the model.
+    @param point_F     The point to be re-expressed.
+    @return point_G    The re-expression of the point in Ground.
+    */
+    SimTK::Vec3 findLocationInGround(const SimTK::State& state,
+                    const SimTK::Vec3& point_F) const;
+
+    /**
+    Take a point located and expressed in this frame (F) and determine
+    its velocity expressed in Ground (G).
+
+    @param state       The state of the model.
+    @param point_F     The point to be re-expressed.
+    @return point_G    The re-expression of the point in Ground.
+    */
+    SimTK::Vec3 findVelocityInGround(const SimTK::State& state,
```

Comment:
This is a bit confusing with `getVelocityInGround` which provides the Spatial Velocity of the Frame. I realize there is an extra argument which is the point, but perhaps we could rename to better differentiate? How about `findPointVelocityInGround` and similarly `findPointLocationInGround` and `findPointAccelerationInGround`? I am also OK with leaving it (already way better!).  Opinions @tkuchida, @chrisdembia @jenhicks?